user-guide/Reference/Soft-launch_options/BrokerGateway_Migration.md

```diff
@@ -0,0 +1,158 @@
+---
+uid: BrokerGateway_Migration
+---
+
+# Migrating to BrokerGateway
+
+Starting from DataMiner 10.5.0/10.5.2, migrating from the SLNet-managed NATS solution (NAS and NATS services) to the BrokerGateway-managed NATS solution (nats-server service) is possible as a [soft-launch feature](xref:SoftLaunchOptions), if the [BrokerGateway](xref:Overview_of_Soft_Launch_Options#brokergateway) soft-launch option is enabled.
+
+From DataMiner 10.6.0 onwards, this migration will be done automatically during a DataMiner upgrade.
+
+## Prerequisites
+
+Before you start the migration, the entire cluster must have been running smoothly for some time. This means that all DataMiner Agents in the cluster must be online and function together for some time. To confirm that the DMS is ready to be migrated, run the [Verify NATS Migration Prerequisites](xref:BPA_NATS_Migration_Prerequisites) BPA.
+
+If the BPA succeeds, the system is all set to migrate.
+
+## Running the migration
```

Comment:
I've adjusted this section to be a numbered procedure, as this is more user-friendly. Could you check if everything still looks OK?